frontend/packages/kubevirt-plugin/src/selectors/immutable/template/combined.ts

```diff
@@ -99,3 +104,24 @@ export const iGetCommonTemplateCloudInit = (tmp: ITemplate) => {
   );
   return iGet(iCloudInitStorage, 'cloudInitNoCloud');
 };
+
+export const iGetDefaultTemplate = (
+  iCommonTemplates: ImmutableMap<string, ITemplate>,
+  os: string,
+): ITemplate =>
+  (iCommonTemplates || ImmutableMap()).valueSeq().find((iTemplate) => {
+    const labels = iGetLabels(iTemplate);
+    return labels && labels.get(getOsLabel(os)) && labels.get(TEMPLATE_DEFAULT_LABEL) === 'true';
+  });
+
+export const getITemplateDefaultFlavor = (template: ITemplate): Flavor =>
+  template &&
+  Flavor.getAll().find(
+    (f) => iGetLabels(template).get(`${TEMPLATE_FLAVOR_LABEL}/${f.getValue()}`) === 'true',
+  );
+
+export const getITemplateDefaultWorkload = (template: ITemplate): WorkloadProfile =>
```

Comment:
the same here. Also this can be generic - not only applying for default

Reply:
https://github.com/openshift/console/pull/7006#discussion_r516639243